frontend/test/__support__/e2e/cypress.js

```diff
@@ -31,3 +32,15 @@ export * from "./helpers/e2e-embedding-helpers";
 export * from "./helpers/e2e-permissions-helpers";
 
 Cypress.on("uncaught:exception", (err, runnable) => false);
+
+const titleToFileName = title => title.replace(/[>]/g, "");
+
+Cypress.on("test:after:run", (test, runnable) => {
+  if (test.state === "failed") {
```

Comment:
For each failure, we are adding the context of the `screenshot` and `video` to the report.